### PR TITLE
Pre start game state

### PR DESCRIPTION
**Pull request by jbernardo95**
This PR includes PR #4

### CODE REVIEW

File: lib/need_for_type/display_window.rb [hunk 1]
```diff
@@ -36,44 +36,37 @@ def standout_mode(option, selected_option)
...
-        self.render_text(c, color, NORMAL)
-        self.render_multiple_text(WHITE, NORMAL) { tail.each { |c| self.addstr(c) } }
+        self.set_render_pos(2, 4)
+        self.render_text("To start playing click ENTER", GREEN, NORMAL)
       end
     end
 
     def render_game_text(text, chars_completed, failed = false)
+      color = failed ? RED : YELLOW
+      head = text[0, chars_completed] || ''
```
Comment (tiagonbotelho) [inline lib/need_for_type/display_window.rb]: I used tail in the previous one but we can use completed/remaining to be more clear

File: lib/need_for_type/states/game.rb [hunk 2]
```diff
@@ -41,12 +43,24 @@ def handle_init_game
...
+    end
+
+    def handle_start_game
+      @display_window.render_start_game
+
+      input = @display_window.get_input
+
+      if input == Curses::Key::ENTER || input == 10
+        @display_window.render_game_text(@text, @chars_completed)
+
```
Comment (tiagonbotelho) [inline lib/need_for_type/states/game.rb]: no need for this extra empty lines

File: lib/need_for_type/states/game.rb [hunk 3]
```diff
@@ -41,12 +43,24 @@ def handle_init_game
...
+        @start_time = Time.now
+
```
Comment (tiagonbotelho) [inline lib/need_for_type/states/game.rb]: and this

**Comment by tiagonbotelho** [comment]
left a few comments but looks nice you can merge after that

File: lib/need_for_type/display_window.rb (hunk 1 above)
Comment (jbernardo95) [reply lib/need_for_type/display_window.rb]: 👍

**Comment by jbernardo95** [comment]
@tiagonbotelho Fixed and merging.

**Pull request merged**